inc/admin/options-page.php

```diff
@@ -216,6 +266,9 @@ public function alcatraz_validate_options( $input ) {
 		if ( ! empty( $input['youtube_url'] ) ) {
 			$options['youtube_url']   = sanitize_text_field( $input['youtube_url'] );
 		}
+		if ( ! empty( $input['show_social_urls'] ) ) {
```

Comment:
See my note on line 231. This is the current check that isn't allowing the option to be set to 0.